app/Http/Controllers/TicketsController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Ticket;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Input;
+
+class TicketsController extends Controller
+{
+    public function index(){
+        $tickets = Ticket::all();
+        return $tickets;
+    }
+
+    public function show($id){
+        $ticket = Ticket::findOrfail($id);
+        return $ticket;
+    }
+
+    public function create(){
+        return view('tickets.create');
+    }
+
+    public function store(Request $request){
+       $rules = array(
+                'tweet_id' => 'required',
+                'premium'  => 'required',
+                'urgency'  => 'required',
+                'opened_by'=> 'required',
```

Comment:
This shouldnt be in the request in the future as it will be created dynamically with the user logged in as:
Auth::user()->